python/lsst/rucio/register/script.py

```diff
@@ -26,9 +26,9 @@
 import sys
 
 from lsst.daf.butler import Butler
-from lsst.dm.rucio.register.app_parser import AppParser
-from lsst.dm.rucio.register.rucio_interface import RucioInterface
-from lsst.dm.rucio.register.rucio_register_config import RucioRegisterConfig
+from lsst.rucio.register.app_parser import AppParser
+from lsst.rucio.register.rucio_interface import RucioInterface
+from lsst.rucio.register.rucio_register_config import RucioRegisterConfig
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Not related, but the `RUCIO_REGISTER_CONFIG` constant defined below replaces and supersedes the one in `rucio_register_config.py`, which is unused elsewhere.  One of these two should be removed.

Also, it's not clear to me why the message needs to be in its own constant.

Reply:
I removed the `RUCIO_REGISTER_CONFIG` from `rucio_register_config.py` - I think that was left over from refactoring an earlier version.

The message was put into a constant because if it had been put inline, one of the style-checking tools complained.